src/Route/FastPsr7RouteInfo.php

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace Penny\Route;
+
+class FastPsr7RouteInfo implements RouteInfoInterface
+{
+    private $name;
+    private $callable;
+    private $params;
+
+    public static function matched($name, callable $callable, $params = [])
```

Comment:
In my opinion also because a `match` method is more related to the `Router` and not the `matched` route, do you agree?